src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -23,42 +24,50 @@ public static void main(String[] args) {
 		SpringApplication.run(DemoApplication.class, args);
 	}
 
-	final ReplayProcessor<String> latestChange = ReplayProcessor.cacheLast();
+	final Sinks.Many<String> latestChange = Sinks.many().replay().latest();
+	final FluxProcessor<String, String> processor = FluxProcessor.fromSink(latestChange);
 
 	public DemoApplication(WebClient.Builder webClientBuilder) {
-		FluxSink<String> sink = latestChange.sink(FluxSink.OverflowStrategy.LATEST);
-
 		WebClient webClient = webClientBuilder.build();
 
 		webClient.get()
 				.uri("https://stream.wikimedia.org/v2/stream/recentchange")
 				.retrieve()
 				.bodyToFlux(JsonNode.class)
-				.name("recentchange")
+				.name("recentChanges")
 				.metrics()
-				.concatMap(change -> {
-					return processChange(change)
+				.concatMap(change -> processChange(change)
+							.onErrorResume(IllegalStateException.class, __ -> Mono.empty())
+							.onErrorContinue(IllegalArgumentException.class, (t, o) -> System.out.println(t))
 							.name("processing")
-							.metrics()
-							.onErrorResume(IllegalStateException.class, __ -> Mono.empty());
-				})
-				.doOnNext(sink::next)
+							.metrics())
+				.doOnNext(processor::onNext)
 				// Avoid polluting the logs
 				.sample(Duration.ofSeconds(1))
 				.log()
-				.retryWhen(it -> it.delayElements(Duration.ofSeconds(1)))
+				.retry()
```

Comment:
is there a particular reason for removing the backoff?

Reply:
I removed it for test purpose (unlimited retries) and forgot to put it back